client/assets/colors.scss

```diff
@@ -1,5 +1,7 @@
 $colors: (
-  yellow: #FFA300,
-  light: #FFF,
+  yellow: #ffa300,
+  white: #fff,
   dark: #000,
-)
+  bonjour: #e0dcdc,
```

Comment:
### Nitpick

Not sure if this is a big deal, but the name `bonjour` isn't descriptive enough.

Reply:
Haha! Just found out this is a color name, well I just learnt something new.